packages/babel-helpers/src/helpers.js

```diff
@@ -396,7 +396,7 @@ helpers.objectSpread = helper("7.0.0-beta.0")`
 
   export default function _objectSpread(target) {
     for (var i = 1; i < arguments.length; i++) {
-      var source = (arguments[i] != null) ? arguments[i] : {};
+      var source = (typeof arguments[i] === 'object' && arguments[i] != null) ? arguments[i] : {};
       var ownKeys = Object.keys(source);
```

Comment:
it would be better to do this:
```suggestion
      var ownKeys = Object.keys(Object(source));
```

Reply:
Thanks for the review! I have made the suggested change @ljharb, can you take another look?